lib/pure/asyncdispatch.nim

```diff
@@ -165,6 +165,30 @@
 ## ================
 ##
 ## * The effect system (`raises: []`) does not work with async procedures.
+##
+##
+## Multiple async backend support
+## ==============================
+##
+## Thanks to its powerful macro support, Nim allows `async`/`await` to be
+## implemented in libraries with only minimal support from the language - as
+## such, multiple `async` libraries exist, including `asyncdispatch` and
+## `asyncdispatch`, and more may come to be developed in the future.
```

Comment:
"asyncdispatch and asyncdispatch" :)

Reply:
10234b363d492b417987bfb979a91d938d006649